modules/aggregation-layers/typed/aggregation-layer.d.ts

```diff
@@ -0,0 +1,48 @@
+import {
```

Comment:
Hey @ibgreen I think a bunch of files in `/typed` accidentally made it in here. Was that intentional?

Reply:
> Hey @ibgreen I think a bunch of files in `/typed` accidentally made it in here. Was that intentional?

Oh no, what a mess...

We need to revert and reapply;

Reply:
I'm about to push up a revert

Reply:
https://github.com/visgl/deck.gl/pull/9318